packages/fxa-shared/payments/stripe.ts

```diff
@@ -94,6 +94,9 @@ export type StripeHelperConfig = {
     plansCacheTtlSeconds: number;
   };
   redis: any; // TODO
+  contentful?: {
```

Comment:
Is it possible to set contentful as a non-optional? This way you can also get rid of the assigned assertion `!` on line 472 below.

Reply:
This is non-blocking.